.github/workflows/main.yml

```diff
@@ -75,3 +75,5 @@ jobs:
         dub build --compiler=${{ env.DC }}
         dub test  --compiler=${{ env.DC }}
         dub run   --compiler=${{ env.DC }} --single test\issue2051_running_unittests_from_dub_single_file_packages_fails.d
+        $env:DUB="$pwd\bin\dub.exe"
+        cd test; dub --single run-unittest.d; $env:DUB=''; $env:DC=''
```

Comment:
I don't quite follow what those `env` steps are doing ?

Reply:
My understanding is that we're setting `DUB` before, then resetting it after.
I don't see the point of resetting it after since the test ends and it wouldn't affect a new step if we were to add one.
Regarding setting it before, I'd prefer if we use the Github syntax, that is:
```yaml
env:
  DUB: ${{ github.workspace }}\bin\dub.exe
run: [...]
```

Reply:
Frankly speaking when I reviewed my code I didn't see any reasons for that except it was the artifact of the past. I left it just because it didn't hurt in general. But now I'm going to fix that.